omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/ViewEncounterWithHtmlFormPageController.java

```diff
@@ -26,6 +28,19 @@ public void get(@RequestParam("encounter") Encounter encounter,
 
         patient.setPatient(encounter.getPatient());
 
+        AdministrationService administrationService = Context.getAdministrationService();
+        
+        String customPrintModule = administrationService.getGlobalProperty("htmlformentryui.customPrintModule");
+        String customPrintUrl = administrationService.getGlobalProperty("htmlformentryui.customPrintUrl");
+        String customPrintTarget = administrationService.getGlobalProperty("htmlformentryui.customPrintTarget");
```

Comment:
We also should confirm the getGlobalProperty method just retursn an empty string if this global property is not found.

Reply:
I believe it returns null, even when "" was entered in advanced settings. It's legal to check for null in the gsp though, see https://github.com/openmrs/openmrs-module-htmlformentryui/pull/27/files#r374305401 . Do you prefer it always be set to empty string if null is returned from admin service?

Reply:
Sounds good, just wanted to make sure you double-checked!